lib/config/configItems.joi.js

```diff
@@ -16,7 +16,7 @@ const bootstrapServers = joi.object({
 
 const bootstrapCloudBackend = joi.object({
     site: joi.string().required(),
-    type: joi.string().valid('aws_s3', 'azure'),
+    type: joi.string().valid('aws_s3', 'azure', 'gcp'),
```

Comment:
Maybe you could try: `.valid(...constants.replicationBackends)`

Reply:
If that makes sense to bind this to valid replication backends (not sure the use for `bootstrapCloudBackends`